doc/connectivity/bluetooth/bluetooth-shell.rst

```diff
@@ -0,0 +1,357 @@
+
+        uart:~$ bt adv-create conn-scan
+        Created adv id: 0, adv: 0x200022f0
+        uart:~$ bt adv-start
+        Advertiser[0] 0x200022f0 set started
+
+You may notice that with this custom advertiser do not advertise the device name, you need to enable
+it. If we take back from the previous example:
+
+.. code-block:: console
+
+        uart:~$ bt adv-stop
+        Advertiser set stopped
+        uart:~$ bt adv-param conn-scan name
+        uart:~$ bt adv-start
+        Advertiser[0] 0x200022f0 set started
+
+You should now see the name of the device in the advertising data. You can also set the advertising
+data manually by using :code:`bt adv-data`. The following example show how to set the advertiser
+name with it:
+
+.. code-block:: console
+
+        uart:~$ bt adv-create conn-scan
+        Created adv id: 0, adv: 0x20002348
+        uart:~$ bt adv-data 1009426C7565746F6F74682D5368656C6C
+        uart:~$ bt adv-start
+        Advertiser[0] 0x20002348 set started
+
+The data must be formatted according to the Bluetooth Core Specification (see version 5.3, vol. 3,
+part C, 11). In this example, the first octet is the size of the data, the second one is type of
+data, 0x09 being the Complete Local Name and the remaining data are the name in ASCII. So, on the
+other device you should see the name *Bluetooth-Shell*.
+
+Directed Advertising
+====================
+
+It is possible to use directed advertising on the shell if you want to reconnect to a device. The
+following example, demonstrate how to create a directed advertiser. The address is specified right
+after the parameter :code:`directed`. The :code:`low` parameter indicates that we want to use the
+low duty cycle mode. :code:`dir-rpa` is required if the remote device has a security level enabling
+authentication.
+
+.. code-block:: console
+
+        uart:~$ bt adv-create conn-scan directed D7:54:03:CE:F3:B4 random low dir-rpa
+        Created adv id: 0, adv: 0x20002348
+
+After that, you can start the advertiser and then the target device will be able to reconnect.
+
+Extended Advertising
+====================
+
+Let's now have a look at some extended advertising features, to enable extended advertising, use the
+`ext-adv` parameter.
+
+.. code-block:: console
+
+        uart:~$ bt adv-create conn-nscan ext-adv name-ad
+        Created adv id: 0, adv: 0x200022f0
+        uart:~$ bt adv-start
+        Advertiser[0] 0x200022f0 set started
+
+This will create an advertiser, connectable and non scannable.
+
+Filter Accept List
+******************
+
+It's possible to create a list of allowed addresses. That list can be used to
+connect to those addresses automatically, here is how to do it:
+
+.. code-block:: console
+
+        uart:~$ bt fal-add 47:38:76:EA:29:36 random
+        uart:~$ bt fal-add 66:C8:80:2A:05:73 random
+        uart:~$ bt fal-connect on
+
+It will then connect to the first available device. In the example, if both devices are advertising
```

Comment:
Is "It" here "device" or "the shell"? Or something else entirely.

Reply:
The device, I will update that.